src/Core/Auth/Entities/WebAuthnCredential.cs

```diff
@@ -29,4 +33,18 @@ public void SetNewId()
     {
         Id = CoreHelpers.GenerateComb();
     }
+
+    public WebAuthnPrfStatus GetPrfStatus()
+    {
+        if (SupportsPrf && EncryptedUserKey != null && EncryptedPrivateKey != null && EncryptedPublicKey != null)
+        {
+            return WebAuthnPrfStatus.Enabled;
+        }
+        else if (SupportsPrf)
+        {
+            return WebAuthnPrfStatus.Supported;
+        }
+
+        return WebAuthnPrfStatus.Unsupported;
```

Comment:
nit (non-blocking): just a tiny suggestion for readability
```suggestion
        if (!SupportsPrf)
        {
            return WebAuthnPrfStatus.Unsupported;
        }
        if (EncryptedUserKey != null && EncryptedPrivateKey != null && EncryptedPublicKey != null)
        {
            return WebAuthnPrfStatus.Enabled;
        }
        return WebAuthnPrfStatus.Supported;

```

Reply:
Fixed!